x11-libs/agg/agg-2.5.recipe

```diff
@@ -36,71 +34,80 @@ PROVIDES_devel="
 	devel:libagggpc$secondaryArchSuffix = 2.0.4 compat >= 2
 	devel:libaggplatformHaiku$secondaryArchSuffix = 2.0.4 compat >= 2
 	devel:libaggplatformsdl$secondaryArchSuffix = 2.0.4 compat >= 2
-"
+	"
 REQUIRES_devel="
 	agg$secondaryArchSuffix == $portVersion base
-"
+	"
 
 PROVIDES_sdl="
 	agg${secondaryArchSuffix}_sdl = $portVersion
 	lib:libaggplatformsdl$secondaryArchSuffix = 2.0.4 compat >= 2
-"
+	"
 REQUIRES_sdl="
 	agg$secondaryArchSuffix == $portVersion base
 	haiku$secondaryArchSuffix
-	lib:libGL$secondaryArchSuffix
-	lib:libsdl_1.2$secondaryArchSuffix
-"
+	lib:libSDL$secondaryArchSuffix
```

Comment:
This was more correct before.

Reply:
Shouldn't it be like libSDL for 1.* and libSDL2_2.0 to declare them?

Reply:
Searching on haikuports it seems that 185 recipes(?) are using lib:libsdl and 39 are using lib:libsdl_1.2, if things need to be changed it's better to get an agreement on this ;)